### ISSUE TITLE
[Bug]: `SemanticCfgGuidedVisitor.onAccessCall` Not Triggered For Indexing Calls Without Index

### ISSUE THREAD

**Issue by loki259** [flowr-analysis/flowr#1739]
### What happened?

I expected the method to trigger for `v[]` in the following R code, which is not the case currently:
```R
v <- c(1, 2, 3)
v[]
```

### What version of R do you have installed?

4.4.5

### What version of flowR did you use?

v2.2.15 (Jun 2, 2025)

### Related input code

```R

```

### Relevant log output

```Shell

```

### Meta Information

- [ ] The Issue is Urgent
- [ ] I Used the Dockerized Version of flowr
